### ISSUE TITLE
What shape is the data we want to put in the table?

### ISSUE THREAD

**Issue by d8aninja** [Haystackers/nfhell#1]

**Comment by d8aninja** [comment]
@nrvedder take a look at https://github.com/Haystackers/nfhell/tree/master/data/csv/2018.  column names and stats in `summary.txt`

**Comment by nrvedder** [comment]
I like it. What is the source? Is this all plays for 2018 or just a sample?

Also looks like there are four columns in there with empty values. Artifact of the data source?

**Comment by d8aninja** [comment]
@nrvedder yeah I think those blank columns separate 5 tables with the name n-dimension.  in the summary, above the identification of those four empty columns, is the source (http://nflsavant.com/pbp_data.php?year=2018).  that site's About page will show that it's every play for 2018 (they also have each of the previous 5 years).  

main concern from eyeballing this dataset is "OffenseTeam"...
```Most common values:    None (3438x)```

everything else looks pretty good; bet we could back out `OffenseTeam` from `DefenseTeam` (unless it's being `None`d for the sake of Special Teams?)
